taqueria-config.ts

```diff
@@ -34,7 +34,6 @@ export const defaultConfig: Config.t = Config.create({
 	sandbox: {
 		local: {
 			label: 'Local Tezos Sandbox',
-			protocol: 'PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg',
```

Comment:
We can now start a sandbox and deploy to it without protocol? I recall not being able to do that for sandboxes

Reply:
That, that's fixed as part of this PR. If omitted, it will default the latest version supported by the flextesa image, and is represented as the Alpha protocol.